automatix/command.py

```diff
@@ -147,6 +154,7 @@ def _python_action(self) -> int:
         locale_vars.update({
             'AbortException': AbortException,
             'SkipBatchItemException': SkipBatchItemException,
+            'PYVARS': self.env.pyvars,
```

Comment:
I am sure whether PYVARS is a good name? Maybe AUTOVARS?

Using PERSISTENT_VARS also in conditions is not an option? There could be an short cut for this like PVARS in conditions.

Reply:
If the handling of PERSISTENT_VARS seems so complicated, maybe we should work on this, e.g. by providing set/get/delete functions?

Reply:
Yes, that should work as intended. That's probably even better, because we don't introduce another way to store stuff.